app/view/twig/_base/_page-nav.twig

```diff
@@ -13,7 +13,11 @@
 
 {% set page_hasnav = true %}
 
-{% set page_nav = block('page_nav')|default('NO NAV') %}
+{% if block('page_nav') is defined %}
+    {% set page_nav = block('page_nav') %}
+{% else %}
+    {% set page_nav = 'NO NAV' %}
```

Comment:
> This could still be a ternary operation:

Ternary works for me. 

> Also doesn't |default skip the defined check?

Wasn't happening any longer in `master` on Twig 2.x-dev … I did look into the code upstream at the time, but yeah … would have to again … 'cause :steam_locomotive: :boom: